### DISCUSSION TITLE
What’s new in Budibase: March 2023

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#10220] [Announcements]
# Budibase March Product Update
**Microsoft/Github SSO | Command palette | Custom branding | Error states | Google Sheets V2 integration**

---

![Frame 1 (7) (1)](https://user-images.githubusercontent.com/49767913/230025080-f88bfbee-c7f8-449d-8324-d2044c02ffb0.png)

## Microsoft/Github Sign In
Sign up and sign in to Budibase using Microsoft Authentication, which now includes Github! You can configure this feature for your self-hosted installation, or use Budibase Cloud where it's already configured out of the box.

[Visit the self-hosted documentation →](https://docs.budibase.com/docs/sso-with-azure-ad)

---
---

![Frame 5 (1) (1)](https://user-images.githubusercontent.com/49767913/230025343-cf4a5c24-30c7-402d-b1ae-630d3d72fade.png)

## Command Palette
Hit the Cmd+K (Mac) or Ctrl+K (Windows) keyboard shortcut to find everything you need within Budibase and run popular commands. Instead of searching and clicking around the editor to perform common actions, you can now use the Command Palette to build faster than ever before.

If you have more ideas for the command palette, please let us know in the comments..

[Visit the documentation →](https://docs.budibase.com/docs/command-palette)

---
---

![Frame 3 (16) (1)](https://user-images.githubusercontent.com/49767913/230025712-1f67edc5-72a1-4f46-bb5e-fd9611cd72c4.png)

## Custom branding (Premium feature)
White-label your Budibase instance and apps. With this premium feature, you can replace/remove the following items:

- Budibase branding within apps
- Logo and content within your login page
- Your app’s favicon, title, and meta data
- Budibase branding within emails

[Visit the documentation →](https://docs.budibase.com/docs/branding)

---
---

### Google Sheets v2 integration
We made updates to the Google Sheets connector, fixing filtering and sorting issues. Expect a more stable and improved experience when using Google Sheets with Budibase.

### Component Error States
We improved the design UI to notify you when your app is not configured correctly and provide call to actions to add components and fix your app in real-time.

**Comment by josh99smith** [comment]
The new update and direction of the proejct is amazing, incredbile work gents! I am bit worried about the monitization model though. We are a non-profit and paying $400/month to unlock the unbranded options is just not realistic or affordable for us :( Wish there was another way, we are self-hosted but I still certainly beleive and understand being compensated for your work. You deserve it!
